src/knx/application_program_object.cpp

```diff
@@ -47,3 +47,26 @@ uint32_t ApplicationProgramObject::getInt(uint32_t addr)
 {
     return ::getInt(TableObject::data() + addr);
 }
+
+double ApplicationProgramObject::getFloat(uint32_t addr, ParameterFloatEncodings encoding)
+{
+    uint8_t buffer[8];
+    switch (encoding)
+    {
+        case Float_Enc_DPT9:
+            memcpyInverted(buffer, TableObject::data() + addr, 2);
+            return Decode_DPT9_Float(*(uint16_t*)buffer);
```

Comment:
can't https://github.com/thelsing/knx/blob/edb9b6b52670be1d6e5a30f261f790a5d090a1d5/src/knx/dptconvert.cpp#L1716 or some other function from dptconvert.cpp be reused here?

Reply:
Sure! my bad. I didn't see those functions. 
a question:
for IEEE754 Double (64bit), should I add "float64FromPayload" and "float64ToPayload" functions to "dptconvert.cpp" ? considering the fact that its not used in any DataPoint?

Reply:
Yes I think this is the best place to put them.